docs/api/data-models/translation-bridge-initialization-request.md

```diff
@@ -0,0 +1,15 @@
+# TranslationBridgeInitializationRequest
+
+Field | Type | Mandatory | Description
+--- | --- | --- | ---
+bridgeId | [TranslationBridgeID](../primitives.md#translationbridgeid) | yes | The given identifier of the intended translation bridge.
+operation | [ServiceOperationName](../primitives.md#serviceoperationname) | yes | The target service-operation.
+inputInterface | [InterfaceName](../primitives.md#interfacename) | yes | The name of the interface template to be translated.
+targetInterface | [InterfaceName](../primitives.md#interfacename) | yes | The name of the interface template of the target serviceoperation.
+targetInterfaceProperties | [Metadata](./metadata.md) | yes | Interface specific data of the target service-operation.
+inputDataModelRequirement | [DataModelID](../primitives.md#datamodelid) | no | The data model identifier of what the consumer can provide as the target serviceoperation input.
```

Comment:
no -> no (yes) 

In the description I'd explain that this is not necessary if there is no input payload or there is no need to translate the input payload. (And something similar to the output as well).